src/app/static/question-filter/question-filter.component.html

```diff
@@ -22,13 +22,14 @@
         [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
         [matChipInputAddOnBlur]="addOnBlur"
         (matChipInputTokenEnd)="add($event)"
+        (keydown.enter)="createFilteredQuestionsUri()"        
       />
     </mat-chip-list>
 
-    <!-- Angular Material, implementing autocomplete on chips -->
+    <!-- Angular Material, implementing autocomplete on chips (keyup.enter)="selected($event)" -->
```

Comment:
why add this to the comment?

Reply:
Aisha accidentally left a reference for herself in a comment.